File: multiples_of_3_and_5.js
var sum = 0;

for(i = 1; i < 1000; i++) {
	if ((i % 3 === 0) || (i % 5 === 0)) {
		sum += i;
	}
}

console.log("The sum of all multiples of 3 and 5 below 1000 is " + sum);

File: multiples_of_3_and_5.rb
# Ruby version of solution

sum = 0
i = 1

while i < 1000
	if((i % 3 == 0) || (i % 5 == 0))
		sum += i
	end
	i += 1
end

puts "The sum of all multiples of 3 and 5 below 1000 is #{sum}"